presto-main-base/src/main/java/com/facebook/presto/metadata/MetadataManager.java

```diff
@@ -1479,6 +1489,23 @@ public void addConstraint(Session session, TableHandle tableHandle, TableConstra
         metadata.addConstraint(session.toConnectorSession(connectorId), tableHandle.getConnectorHandle(), tableConstraint);
     }
 
+    @Override
+    public String normalizeIdentifier(Session session, String catalogName, String identifier)
+    {
+        long startTime = System.nanoTime();
+        String normalizedString = identifier.toLowerCase(ENGLISH);
+        Optional<CatalogMetadata> catalogMetadata = getOptionalCatalogMetadata(session, transactionManager, catalogName);
+        if (catalogMetadata.isPresent()) {
+            ConnectorId connectorId = catalogMetadata.get().getConnectorId();
+            ConnectorMetadata metadata = catalogMetadata.get().getMetadataFor(connectorId);
+            normalizedString = metadata.normalizeIdentifier(session.toConnectorSession(connectorId), identifier);
+            session.getRuntimeStats().addMetricValue(GET_IDENTIFIER_NORMALIZATION_TIME_NANOS, NANO, System.nanoTime() - startTime);
+            return normalizedString;
```

Comment:
you could remove this return and the above session.getRuntimeStats().addMetricValue, since you are just assigning `normalizedString`

Reply:
Also just curious how useful this metric might be - are there some normalization that could take considerable time?

Reply:
> You could add a method to the base class like loadTableAndNormalizeName that just calls normalizeIdentifier then loadTable in the subclass. You would just have to update the existing calls to loadTable.

Regarding `loadTable` method clean up in tests, I was thinking I could do that in different PR since that change is not related? Would it be Ok?

Reply:
It's not a big deal, it's only a few repeated lines and I thought it would a little cleaner but either way is fine.